src/solidity-frontend/solidity_grammar.cpp

```diff
@@ -269,11 +291,11 @@ ElementaryTypeNameT get_elementary_type_name_t(const nlohmann::json &type_name)
   if(typeString.find("int_const") != std::string::npos)
   {
     /**
-     * For Literal, their typeString is like "int_const 100".
-     * There is no additional type info (bitsize, signed/unsigned),
-     * This means it will require additional type info from the parent
-     * expr to create an internal type.
-     */
+       * For Literal, their typeString is like "int_const 100".
+       * There is no additional type info (bitsize, signed/unsigned),
+       * This means it will require additional type info from the parent
+       * expr to create an internal type.
+       */
```

Comment:
This formatting change looks wrong. If performed by clang-format, some earlier definition caused it to believe indentation would be one level higher.

Reply:
Will fix. Thanks for pointing this out.